src/app/vault/ciphers.component.ts

```diff
@@ -44,6 +44,11 @@ export class CiphersComponent extends BaseCiphersComponent implements OnDestroy
     actionPromise: Promise<any>;
     userHasPremiumAccess = false;
 
+    organizationNames = new Map<string, string>();
+    sortedDescending = true;
+    sortBy = '' as string;
```

Comment:
Instead of `as` use `: type`
```suggestion
    sortBy: string = '';
```